### PR TITLE
Ben - LB120.LC_GIS_R_LTgis_Yh_GLU

### PR DESCRIPTION
**Pull request by bpbond**
Addresses #211

### CODE REVIEW

File: R/zchunk_LB120.LC_GIS_R_LTgis_Yh_GLU.R [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    # LAND COVER FOR LAND ALLOCATION
+    # Aggregate into GCAM regions and land types
+    # Part 1: Land cover by GCAM land category in all model history/base years
+    # Collapse land cover into GCAM regions and aggregate land types
+    L100.Land_type_area_ha %>%
+      group_by(GCAM_region_ID, Land_Type, year, GLU) %>%
+      summarise(Area_bm2 = sum(Area_bm2)) %>%
+      ungroup %>%
+      # Missing values should be set to 0 before interpolation, so that in-between years are interpolated correctly
+      # We dot his because Alan Di Vittorio (see sources above) isn't writing out all possible combinations of
```
Comment (kvcalvin) [inline R/zchunk_LB120.LC_GIS_R_LTgis_Yh_GLU.R]: Small edit: "We do this" instead of "We dot his"

File: R/zchunk_LB120.LC_GIS_R_LTgis_Yh_GLU.R [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+      L120.LC_bm2_R_LT_Yh_GLU
+
+    # Subset the land types that are not further modified
+    L120.LC_bm2_R_UrbanLand_Yh_GLU <- filter(L120.LC_bm2_R_LT_Yh_GLU, Land_Type == "UrbanLand")
+    L120.LC_bm2_R_Tundra_Yh_GLU <- filter(L120.LC_bm2_R_LT_Yh_GLU, Land_Type == "Tundra")
+    L120.LC_bm2_R_RckIceDsrt_Yh_GLU <- filter(L120.LC_bm2_R_LT_Yh_GLU, Land_Type == "RockIceDesert")
+
+    # LAND COVER FOR CARBON CONTENT CALCULATION
+    # Compile data for land carbon content calculation on unmanaged lands
+    # Note: not just using the final year, as some land use types may have gone to zero over the historical period.
```
Comment (kvcalvin) [inline R/zchunk_LB120.LC_GIS_R_LTgis_Yh_GLU.R]: I'm not sure I understand this at all. Are we just averaging land over the historical period?

Comment (bpbond) [inline R/zchunk_LB120.LC_GIS_R_LTgis_Yh_GLU.R]: Yes, that's what's happening in the code. @pkyle can you provide any more context/reasoning to improve the comments here? Thanks.

Comment (pkyle) [inline R/zchunk_LB120.LC_GIS_R_LTgis_Yh_GLU.R]: The HYDE data are provided in increments of 10 years, so any GCAM model time period or carbon cycle year that ends in a 5 (e.g., 1975) is computed as an average of surrounding time periods. For most of the years that we want, we aren't doing any real averaging or interpolation.

**Pull request merged**